data-prepper-plugins/ruby-processor/src/test/java/org/opensearch/dataprepper/plugins/processor/ruby/RubyProcessorE2ETest.java

```diff
@@ -0,0 +1,4 @@
+package org.opensearch.dataprepper.plugins.processor.ruby;
+
+public class RubyProcessorE2ETest {
```

Comment:
I did not mean to make this a new file and thought I deleted it from the commit — I will remove this on next revision.